bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,329 @@
+            await ctx.send("Sorry, the poll has started! You can try and participate in the next round though!")
+
+    @commands.Cog.listener()
+    async def on_reaction_add(self, reaction: discord.Reaction, user: discord.User) -> None:
+        """Ensures that each user adds one and only one reaction."""
+        try:
+            if reaction.emoji in self.emojis_val.keys() and reaction.message.id in self.messages.keys():
+                msg = reaction.message
+                msg: Message
+                # create a custom counter
+                counter = {}
+                for r in msg.reactions:
+                    async for user in r.users():
+                        if user in counter:
+                            counter[user] += 1
+                        else:
+                            counter[user] = 1
+
+                if counter[user] > 1 and user != self.bot.user:  # If the user has more than one reaction
+                    await user.send("Sorry, you have already added a reaction, please remove your reaction and try \
+again")
+                    return await reaction.remove(user)  # removes user's reaction
+        except RuntimeError:  # The dictionary was changed in between
+            pass
+
+    @spook_name_rate.command(name='delete')
+    async def delete_word(self, ctx: Context) -> None:
+        """Delete's the user's word."""
+        if not self.poll:
+            for message_id in self.messages.keys():
+                data = self.messages[message_id]
+                if ctx.author == data['author']:
+                    del self.messages[message_id]  # delete the data
+                    if self.messages.get(message_id) is None:  # Make sure it does not exist
+                        # display if deletion successful
+                        return await ctx.send(f'Message deleted successfully ({data["word"]!r})!')
+                    else:
+                        # display if deletion unsuccessful
+                        return await ctx.send("Oops, there was some error, please try again!")
+            return ctx.send("But you don't have an entry... :eyes: Type `spooknamerate `")
+        else:
+            await ctx.send("You can't delete your word since the poll has already started!")
+
+    @tasks.loop(hours=24.0)
+    async def announce_word(self) -> None:
+        """Announces the name needed to spookify every 24 hours and the winner of the previous game."""
+        channel = await self.get_channel()
+
+        if self.first:
+            for message in ["Okkey... Welcome to Spook Name Rate! It's a relatively simple game.",
+                            "Everyday, a random name will be sent in `#seasonalbot-commands`",
+                            f"And you need to try and spookify it! Register your word using \
+`{self.bot.command_prefix}spooknamerate add spookified_name`"]:
+
+                await channel.send(message)
+                await asyncio.sleep(1)
```

Comment:
What does this do?

Reply:
It just puts a pause for one second after sending each message. Shall I remove it?

Reply:
I have removed it and fixed it as you have mentioned in the comment below.